inc/sainexthop.h

```diff
@@ -229,6 +229,17 @@ typedef enum _sai_next_hop_attr_t
      */
     SAI_NEXT_HOP_ATTR_DECREMENT_TTL,
 
+    /**
+     * @brief Encap tunnel-id for MPLS PUSH
+     *
+     * @type sai_object_id_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @objects SAI_OBJECT_TYPE_TUNNEL
+     * @allownull true
+     * @condition SAI_NEXT_HOP_ATTR_TYPE == SAI_NEXT_HOP_TYPE_MPLS
+     */
+    SAI_NEXT_HOP_ATTR_MPLS_ENCAP_TUNNEL_ID,
```

Comment:
What tunnel attributes do you expect to have besides TTL and EXP mode?

Reply:
right now only TTL and EXP mode, I considered another attribute of Label stack which is already part of "next hop". Tunnel Label Stack is pushed on top of Next Hops label stack. But I wasn't sure if it made sense to partition label stack in that way.